src/store/localStorage.js

```diff
@@ -0,0 +1,9 @@
+import localStoragePackage from 'localStorage';
+
+let localStorage = localStoragePackage; // eslint-disable-line
```

Comment:
this one 
```
/Users/esbb48/project/self/Kizuna/src/store/localStorage.js
  3:1  error  Exporting mutable 'let' binding, use 'const' instead  import/no-mutable-exports
```

AND I change to use `// eslint-disable-next-line import/no-mutable-exports`